cacheops/query.py

```diff
@@ -248,6 +248,10 @@ def clone(self, klass=None, setup=False, **kwargs):
             clone._cloning = self._cloning - 1 if self._cloning else 0
             return clone
 
+    def _send_post_lookup_signal(self, hit_cache):
+        model_path = "%s.%s" % (self.model.__module__, self.model.__name__)
+        post_lookup.send(sender='cacheops.iterator', model=model_path, hit_cache=hit_cache)
```

Comment:
I wanted to provide some context about where the signal was emitted from. Maybe passing  `self.model` here as sender would be better.